test/usecases/log_in_use_case_test.dart

```diff
@@ -10,6 +10,7 @@ void main() {
   group('LoginUseCaseTest', () {
     late MockAuthenticationRepository mockAuthenticationRepository;
     late MockSecureStorage mockSecureStorage;
+    late MockSharedPreferencesStorage mockSharedPreferencesStorage;
```

Comment:
@Shayokh144 I think we could update the test to verify that `_sharedPreferences.saveLoginStatus(true);` was called 💭